src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    @Override
+    public String toString() {
+        return "Lottery{color = " + getRandomBall().getColor()
+                + ", number = " + getRandomBall().getNumber() + "}";
+    }
+
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        Random random = new Random();
+        int value = random.nextInt(100);

```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants